sdk/translation/Azure.AI.Translation.Document/src/DocumentTranslationOperation.cs

```diff
@@ -299,7 +299,7 @@ private async ValueTask<Response> UpdateStatusAsync(bool async, CancellationToke
                     }
                     else if (update.Value.Status == DocumentTranslationStatus.ValidationFailed)
                     {
-                        DocumentTranslationError error = update.Value.Error;
+                        DocumentTranslationError error = (DocumentTranslationError)update.Value.Error;
```

Comment:
I'm a bit confused. Why do we need this cast?

Reply:
`update.Value.Error` is nullable, but at this point we know there is content there, so we do the cast to a non-nullable type. 
I could always check that the value is not null and then assign the value if that will be more clear

Reply:
I think the current approach is good enough. Looks good to me!